packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/util.ts

```diff
@@ -0,0 +1,55 @@
+// The contents of this file were taken from the AWS CDK provider framework.
+// https://github.com/aws/aws-cdk/blob/c52ff08cfd1515d35feb93bcba34a3231a94985c/packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/util.ts
```

Comment:
Basically the same as `utils.ts` in the CDK custom provider framework.
Modifications:
- replaced `log` function with the `log` function already existing in our Lambda code.